xml/decoders/SoundTraxx_Tsu_SoundCar_Walthers.xml

```diff
@@ -234,8 +234,8 @@
     <programming direct="yes" paged="yes" register="yes" ops="yes">
         <capability>
             <name>Indexed CV access</name>
-            <parameter name="PI">31</parameter>
-            <parameter name="SI">32</parameter>
```

Comment:
The index CV changes have come from an outdated version. Please revert those particular changes.

Reply:
reverted as well